### DISCUSSION TITLE
18+ rating label (VNG)

### DISCUSSION THREAD

**Discussion by yurixahri** [PenguLoader/PenguLoader#8] [Q&A] [answered]
Do you now where the 18+ rating label appear in the client (vn server) ? i've checked all the wad.client files and there is no sign of it, or it's just doesn't there a whole time ?

**Comment by nomi-san** [comment, marked as answer]
Just use DevTools to inspect it.

![image](https://user-images.githubusercontent.com/38210249/212470847-dc87e46e-11e0-4c3c-ba44-3bf02bb60896.png)

Found the class name: **"vng-age-rating ember-view"**.

Just put this code into the console:
```js
document.querySelector('div[class="vng-age-rating ember-view"]').style.display = 'none';
```

You can write your own plugin to automate this task.

**Reply by yurixahri** [reply]
so basically the client is pure web ?

**Reply by nomi-san** [reply]
Yesh, League Loader can help you.

**Reply by yurixahri** [reply]
is the loader support jquery ?

**Reply by nomi-san** [reply]
To use jQuery, just refer to the section **Native ESM supports** in README.

```js
(async function test() {
  const { default: $ } = await import('https://cdn.skypack.dev/jquery');
  // do something
})();
```

**Reply by yurixahri** [reply]
ok thanks for your answers, i'll go explore things and see if there's any problem.
